data/source/gj_1111.json

```diff
@@ -71,6 +71,9 @@
         {
             "other_name": "LHS 248"
         },
+        {
+            "other_name": "LP  258-28"
+        },
         {
```

Comment:
Wrong match. LP 258-28 is LHS 2023.  https://simbad.u-strasbg.fr/simbad/sim-id?Ident=LP++258-28&NbIdent=1&Radius=2&Radius.unit=arcmin&submit=submit+id